BLSQ/iaso
# IA-2234: use AlertModal from bluesquare-components

The component was generic enough to be moved to the library

Related JIRA tickets : IA-2234

## Self proofreading checklist


## Doc
NA

## Changes

- Delete AlertModal in iaso
- update imports in RoudDatesHistoryModal and NotificationImportDetailModal

## How to test

update the dependency for bluesquare-components to this branch: https://github.com/BLSQ/bluesquare-components/tree/IA-2234_move_AlertModal
launch a worker

Go to Polio > Campaigns : open a campaign then check the history for a round
Go to Polio > Virus notification: start an import
Go to tasks, click on the action button for the polio notification task: the modal should open

## Print screen / video

There should be no visible change

## Notes

Requires: https://github.com/BLSQ/bluesquare-components/pull/145

IMPORTANT: package-lock.json should be updated before merging!
